benchmarks/wpt_bench.cpp

```diff
@@ -7,7 +7,11 @@ double url_examples_bytes{};
 
 std::vector<std::pair<std::string, std::string>> url_examples;
 
-size_t init_data(const char *source) {
+enum {
+  ALL_URLS = -1,
```

Comment:
Can you also add a documentation on how to run the benchmarks?

Reply:
@anonrig I pushed another commit which I expect might fix this build issue.